src/clj/rems/auth/oidc.clj

```diff
@@ -51,9 +51,15 @@
                        ;; providers differ in what they give us
                        :commonName (some id-data [:name :unique_name :family_name])
                        :mail (:email id-data)}
-        extra-attributes (select-keys id-data (map (comp keyword :attribute) (:oidc-extra-attributes env)))]
+        extra-attributes (select-keys id-data (map (comp keyword :attribute) (:oidc-extra-attributes env)))
+
+        user-info (when-let [url (:userinfo_endpoint oidc-configuration)]
```

Comment:
Why the underscore in the key?

Reply:
The `oidc-configuration` is read from `https://{oidc-domain}/.well-known/openid-configuration`, and we don't do `snake_case` to `kebab-case` conversion for the keys. I think it's good to use the standard keys here instead of changing their format.